packages/actor-rdf-resolve-hypermedia-sparql/lib/ActorRdfResolveHypermediaSparql.ts

```diff
@@ -53,6 +62,10 @@ export interface IActorRdfResolveHypermediaSparqlArgs extends IActorRdfResolveHy
    * @default {false}
    */
   forceHttpGet: boolean;
+  /**
+   * A mediator for creating binding context merge handlers
+   */
+  mediatorMergeHandlers: MediatorMergeBindingFactory;
```

Comment:
Can we rename this mediator to `mediatorMergeBindingsContext`? (also in other places)